WordPress/src/main/java/org/wordpress/android/ui/accounts/login/Login2FaFragment.java

```diff
@@ -116,6 +117,7 @@ protected void onHelp() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        ((WordPress) getActivity().getApplication()).component().inject(this);
```

Comment:
The injection in the base class is more tricky because it's a generic class. That's why I moved the injection up.